app/cdap/components/CaskWizards/AddNamespace/index.js

```diff
@@ -112,21 +112,49 @@ export default class AddNamespaceWizard extends Component {
       namespaceId: newNamespaceId,
     });
     let buttonLabel = T.translate(`${PREFIX}.callToAction`, { namespaceId: newNamespaceId });
-    let linkLabel = T.translate('features.Wizard.GoToHomePage');
     this.setState({
       successInfo: {
         message,
         buttonLabel,
         buttonUrl: window.getAbsUIUrl({
           namespaceId: newNamespaceId,
         }),
-        linkLabel,
-        linkUrl: `${window.getAbsUIUrl({
-          namespaceId: currentNamespaceId,
-        })}/control`,
+        links: this.getLinks(newNamespaceId, currentNamespaceId),
       },
     });
   }
+
+  isNamespacedServiceAccountsEnabled() {
+    return window.CDAP_CONFIG.featureFlags['feature.namespaced.service.accounts.enabled'] === 'true'
+      ? true
+      : false;
```

Comment:
The `? true : false` part is unnecessary, as the `window.CDAP_CONFIG.featureFlags['feature.namespaced.service.accounts.enabled'] === 'true'` expression evaluates to a boolean.

Reply:
yes, missed it.. will change